spec/features/ua_user_update_cart_spec.rb

```diff
@@ -37,7 +37,7 @@
 
     #======update quantities=====
     within_table("cart_table") do
-    fill_in 'quantity', :with => 2
+    find('#quantity',:match => :first).find(:xpath, 'option[2]').select_option
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 (not 0) spaces for indentation.<br>Space missing after comma.